cg/services/slurm_upload_service/slurm_upload_service.py

```diff
@@ -0,0 +1,42 @@
+from cg.apps.tb.api import TrailblazerAPI
+from cg.apps.tb.models import TrailblazerAnalysis
+from cg.constants.priority import SlurmQos
+from cg.models.slurm.sbatch import Sbatch
+from cg.services.slurm_service.slurm_service import SlurmService
+from cg.services.slurm_upload_service.slurm_upload_config import SlurmUploadConfig
+from cg.services.slurm_upload_service.utils import get_quality_of_service
+
+UPLOAD_MEMORY = 1
+UPLOAD_MAX_HOURS = 24
+EXCLUDED_COMPUTE_NODES = "--exclude=gpu-compute-0-[0-1],cg-dragen"
```

Comment:
have a look at cg/constants/slurm.py we can declare these constants there I think